### PR TITLE
chore: filter out empty requirements

### LINKED ISSUE #1813: chore: remove default eligibility requirement on round creation process
**Overview**
What's the goal

**Details**
Say more

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

fixes #1813

### CODE REVIEW

File: packages/round-manager/src/context/round/CreateRoundContext.tsx [hunk 1]
```diff
@@ -139,6 +139,14 @@ const _createRound = async ({
   try {
     datadogLogs.logger.info(`_createRound: ${round}`);
 
+    if (roundMetadataWithProgramContractAddress.eligibility) {
+      roundMetadataWithProgramContractAddress.eligibility.requirements = roundMetadataWithProgramContractAddress.eligibility?.requirements.filter(
+        obj => obj.requirement != ""
+      );
+    }
+
+    applicationQuestions.applicationSchema.requirements
```
Comment (DanieleSalatti) [inline packages/round-manager/src/context/round/CreateRoundContext.tsx:148]: leftover?

Comment (thelostone-mc) [reply packages/round-manager/src/context/round/CreateRoundContext.tsx:148]: derp 📦

Comment (thelostone-mc) [inline packages/round-manager/src/context/round/CreateRoundContext.tsx:148]:
```suggestion
```

File: packages/round-manager/src/context/round/CreateRoundContext.tsx [hunk 2]
```diff
@@ -139,6 +139,13 @@ const _createRound = async ({
   try {
     datadogLogs.logger.info(`_createRound: ${round}`);
 
+    if (roundMetadataWithProgramContractAddress.eligibility) {
+      roundMetadataWithProgramContractAddress.eligibility.requirements = roundMetadataWithProgramContractAddress.eligibility?.requirements.filter(
+        obj => obj.requirement != ""
+      );
+    }
+
+
```
Comment (thelostone-mc) [inline packages/round-manager/src/context/round/CreateRoundContext.tsx:148]:
```suggestion
```

**Comment by michellema1208** [comment]
@thelostone-mc - is there a way for Product or Design to test this via UI? If not, I defer to Engineering reviews for this change

**Comment by michellema1208** [comment]
@thelostone-mc it looks like the build on this is still queued, so am not able to test atm!

**Comment by thelostone-mc** [comment]
> @thelostone-mc - is there a way for Product or Design to test this via UI? If not, I defer to Engineering reviews for this change


This would be an engineering review

**Review by hmrtn — APPROVED** [review]

**Review by michellema1208 — APPROVED** [review]

**Pull request merged**